content/applications/finance/accounting/reporting.rst

```diff
@@ -19,136 +18,135 @@ countries :
+a particular date.
+
+.. image:: reporting/reporting-balance-sheet.png
+   :alt: Balance sheet report of Odoo.
 
-.. image:: reporting/main_reports09.png
+.. _reporting/profit-and-loss:
 
 Profit and Loss
 ---------------
 
-The **Profit and Loss** report (or **Income Statement**) shows your
-organisation's net income, by deducting expenses from revenue for the
-report period.
+The **Profit and Loss** report (or **Income Statement**) shows your company's net income by
+deducting expenses from revenue for the report period.
 
-.. image:: reporting/main_reports10.png
+.. image:: reporting/reporting-profit-and-loss.png
+   :alt: Profit and Loss report of Odoo
+
+.. _reporting/executive-summary:
 
 Executive Summary
 -----------------
 
-The **Executive Summary** allows for a quick look at all the important
-figures you need to run your company.
+The **Executive Summary** allows for a quick look at all the important figures to run your company.
 
-In very basic terms, this is what each of the items in this section is
-reporting :
+In basic terms, this is what each item in the following section reports:
 
 - **Performance:**
     - **Gross profit margin:**
-        The contribution each individual sale made
-        by your business less any direct costs needed to make those sales
-        (things like labour, materials, etc).
+        The contribution of each individual sale made by your business **minus** any direct costs
+        needed to make those sales (labour, materials, etc).
     - **Net profit margin:**
-        The contribution each individual sale made by
-        your business less any direct costs needed to make those sales,
-        as well as any fixed overheads your company has (things like
-        rent, electricity, taxes you need to pay as a result of those
-        sales).
+        The contribution of each individual sale made by your business **minus** any direct costs
+        needed to make those sales *and* fixed overheads your company has (electricity, rent, taxes
+        to be paid as a result of those sales, etc).
     - **Return on investment (p.a.):**
-        The ratio of net profit made, to the
-        amount of assets the company used to make those profits.
+        The ratio of net profit made to the amount of assets the company used to make those profits.
 - **Position:**
     - **Average debtor days:**
-        The average number of days it takes your
-        customers to pay you (fully), across all your customer invoices.
+        The average number of days it takes your customers to (fully) pay you across all your
+        customer invoices.
     - **Average creditor days:**
-        The average number of days it takes you to
-        pay your suppliers (fully) across all your bills.
+        The average number of days it takes you to (fully) pay your suppliers across all your bills.
     - **Short term cash forecast:**
```

Comment:
```suggestion
    - **Short-term cash forecast:**
```

Reply:
It's without a hyphen in Odoo (I'm aware it's a typo)